test/lib/ansible_test/_util/target/setup/bootstrap.sh

```diff
@@ -144,19 +144,36 @@ bootstrap_remote_freebsd()
         jinja2_pkg="py${python_package_version}-jinja2"
         cryptography_pkg="py${python_package_version}-cryptography"
         pyyaml_pkg="py${python_package_version}-yaml"
+        packaging_pkg="py${python_package_version}-packaging"
 
         # Declare platform/python version combinations which do not have supporting OS packages available.
         # For these combinations ansible-test will use pip to install the requirements instead.
         case "${platform_version}/${python_version}" in
+            14.0/3.9)
+				# defaults above 'just work'TM
+                ;;
+            14.0/3.10)
+                cryptography_pkg=""  # not available
+                jinja2_pkg=""  # not available
+                pyyaml_pkg=""  # not available
+                ;;
+            14.0/3.11)
+                cryptography_pkg=""  # not available
+                jinja2_pkg=""  # not available
+                pyyaml_pkg=""  # not available
+                ;;
             *)
+				# just assume nothing is available
```

Comment:
Indent with spaces instead of tabs.

```suggestion
                # just assume nothing is available
```